src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -505,13 +505,21 @@ internal bool IsExpressionBodied
 
         private void CheckInitializer(
             bool isAutoProperty,
+            bool isInterface,
+            bool isStatic,
             Location location,
             DiagnosticBag diagnostics)
         {
-            if (!isAutoProperty)
+            if (isInterface && !isStatic)
+            {
+                diagnostics.Add(ErrorCode.ERR_InstancePropertyInitializerInInterface, location, this);
+            }
+            else if (!isAutoProperty)
             {
                 diagnostics.Add(ErrorCode.ERR_InitializerOnNonAutoProperty, location, this);
+
             }
+
         }
```

Comment:
formatting nit: unnecessary empty line above

Reply:
Missed that in linting, fixed now. Thanks!